integration/screen_widgets_test.go

```diff
@@ -45,6 +45,11 @@ func TestWidgets(t *testing.T) {
 						Type:    datadog.String("dashed"),
 						Width:   datadog.String("thin"),
 					},
+					Metadata: map[string]datadog.TileDefMetadata{
```

Comment:
Is this just testing that a screenboard widget object can be created without error?

Reply:
If you look at the loop at the end of this test method, it does this:

* Creates an empty screeboard
* Updates it with a definition of an individual widget (=> so creates the widget)
* Reads the whole screenboard back and verifies that it got back exactly what it posted

So this tests both creating and reading back the values.